keysManagement/managedPeersHolder.go

```diff
@@ -299,6 +300,23 @@ func (holder *managedPeersHolder) GetManagedKeysByCurrentNode() map[string]crypt
 	return allManagedKeys
 }
 
+// GetLoadedKeysByCurrentNode returns all keys that were loaded and will be managed by this node
+func (holder *managedPeersHolder) GetLoadedKeysByCurrentNode() [][]byte {
+	holder.mut.RLock()
+	defer holder.mut.RUnlock()
+
+	allLoadedKeys := make([][]byte, 0, len(holder.data))
+	for pk := range holder.data {
+		allLoadedKeys = append(allLoadedKeys, []byte(pk))
+	}
+
+	sort.Slice(allLoadedKeys, func(i, j int) bool {
+		return string(allLoadedKeys[i]) < string(allLoadedKeys[j])
+	})
```

Comment:
extract this logic into a separate func and used in ManagedKeysByCurrentNode and remove sorting logic from managePeersMonitor?

Reply:
ManagedKeysByCurrentNode has an extra check than GetLoadedKeysByCurrentNode, the one for validators.. thus the common code is just the sorting which can be done separate